apps/apple/FluentUITester/macos/FluentUITester-macOS/ViewController.m

```diff
@@ -0,0 +1,22 @@
+#import "ViewController.h"
+#import "AppDelegate.h"
+
+#import <React/RCTRootView.h>
+
+@implementation ViewController
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  RCTBridge *bridge = [((AppDelegate *)[NSApp delegate])bridge];

```

Comment:
Can we explicitly pass the bridge around instead of assuming time of app delegate? This is the sort of thing that makes it difficult to keep up with Apple updates

Reply:
We should type check it first, but I'm not sure where you're saying we should get it from if not here. I'd rather not stray too far from how RN hooks this up though, at least at first

Reply:
This doesn't seem like it has anything to do with the RN implementation. We should pass an explicit reference to the bridge from the AppDelegate (does it need to be initialized there?) to the ViewController

Reply:
By that I meant this is how React-Native recommends consumers create/manage their content via a bridge. We can stray from it, but we shouldn't without a good reason

Reply:
Without rewriting how RN works, we need the app delegate to conform to an RCT protocol. We still will want to cast the value to be safe, but to simplify things I moved the bridge retrieval logic into the view controller